python/pfs/drp/stella/createRefSpec.py

```diff
@@ -0,0 +1,118 @@
+import os
+import sys
+import argparse
+
+import pfs.drp.stella.findAndTraceAperturesTask as fataTask
+import pfs.drp.stella.createFlatFiberTraceProfileTask as cfftpTask
+import pfs.drp.stella.extractSpectraTask as esTask
+import pfs.drp.stella as drpStella
+import lsst.daf.persistence as dafPersist
+import lsst.afw.image as afwImage
+from lsst.pex.config import Config, ConfigField, ConfigurableField, Field, ListField
+from lsst.pipe.base import Task, Struct, TaskRunner, ArgumentParser, CmdLineTask
+import lsst.daf.persistence.butler as lsstButler
+import numpy as np
+from astropy.io import fits as pyfits
+import matplotlib.pyplot as plt
+
```

Comment:
All of `os`, `sys`, `argparse`, `dafPersist`, `afwImage,` `ConfigField`, `ListField`, `ConfigurableField`, `Task`, `lsstButler`, `plt` are unused!